libs/designsystem/src/lib/helpers/line-clamp-helper.ts

```diff
@@ -0,0 +1,10 @@
+import { Injectable } from '@angular/core';
+
+@Injectable()
+export class LineClampHelper {
+  setLineClamp(el: HTMLElement, maxLines: number, lineHeight: string): void {
+    el.style.setProperty('--line-clamp', `${maxLines}`);
+    el.style.setProperty('--line-height', lineHeight);
+    el.classList.add('kirby-line-clamp');
```

Comment:
We generally use Renderer2 for modifying the DOM...

Reply:
Renderer2 didn't apply the CSS custom properties, but afterwards I was told it should be possible, if an extra parameter is applied. Haven't looked into it yet.

Could this be the reason that it doesn't initially get applied in Safari?

Reply:
Changed it to use Renderer2